module/VuFindTheme/src/VuFindTheme/ResourceContainer.php

```diff
@@ -160,11 +251,43 @@ public function getCss()
     /**
      * Get Javascript files.
      *
+     * @param string $position Position where the files should be inserted
+     * (allowed values are 'header' or 'footer').
+     *
      * @return array
      */
-    public function getJs()
+    public function getJs(string $position=null)
```

Comment:
For some reason, none of the style checkers seem to mind, but I generally put spaces around `=` characters in function definitions. Might be worth changing here for consistency.